onnxruntime/test/testdata/multi_stream_models/simplified_ssd_cpu.json

```diff
@@ -0,0 +1,4 @@
+{
+"type":"DeviceBasedPartitioner",
+"streams":[["conv_0","relu_0","maxpool_0","conv_4","add_0"],["conv_1","relu_1","conv_2","relu_2","conv_3"]]
+}
```

Comment:
Can we document the JSON format somewhere?

Reply:
Exactly, done.